model/span.go

```diff
@@ -189,6 +197,19 @@ func (d *DestinationService) fields() common.MapStr {
 	return common.MapStr(fields)
 }
 
+func (c *Composite) fields() common.MapStr {
+	if c == nil {
+		return nil
+	}
+	var fields mapStr
+	if c.Count > 1 {
+		fields.set("count", c.Count)
+	}
+	fields.set("exact_match", c.ExactMatch)
+	fields.maybeSetMapStr("end", utility.TimeAsMicros(c.End))
```

Comment:
@axw The `end` field is required in the composite object, so this can also be simply `fields.set("end", utility.TimeAsMicros(c.End))`, correct?

Reply:
Yes, unless `c.End` can be zero, in which case `utility.TimeAsMicros(c.End)` will return `nil`. AFAICS we're not currently validating that c.End is > 0.